server/neptune/workflows/activities/main.go

```diff
@@ -31,6 +33,10 @@ const (
 	// terraformPluginCacheDir is the name of the dir inside our data dir
 	// where we tell terraform to cache plugins and modules.
 	TerraformPluginCacheDirName = "plugin-cache"
+
+	// StartToCloseTimeout for PRRevision activities is configured to 30 seconds
+	// Setting the revision setter client timeout to 20 seconds < StartToCloseTimeout
+	RevisionSetterClientTimeout = 20 * time.Second
```

Comment:
That's fair. If the activity times out on the `StartToCloseTimeout`, the request context should get cancelled. I've removed the client timeout so that it defaults to no timeout.